dashboard/app/models/new_feature_feedback.rb

```diff
@@ -0,0 +1,26 @@
+# == Schema Information
+#
+# Table name: new_feature_feedbacks
+#
+#  id         :bigint           not null, primary key
+#  user_id    :integer          not null
+#  form_key   :string(255)      not null
+#  satisfied  :boolean          not null
+#  locale     :string(255)
+#  created_at :datetime         not null
+#
+# Indexes
+#
+#  index_new_feature_feedbacks_on_satisfied  (satisfied)
+#  index_new_feature_feedbacks_on_user_id    (user_id) UNIQUE
+#
+# A simple thumbs up/down feedback form.
+# Intended for use by a front-end banner for new feature feedback.
+class NewFeatureFeedback < ApplicationRecord
+  belongs_to :user
+
+  validates :user_id, uniqueness: true
```

Comment:
Is the intent to allow the same user to leave feedback on multiple features over time? This validation line will stop them after their first feedback. Validating on uniqueness of the (`user_id`, `form_key`) pair might be a better way to go. The unique index on the database table would also need to be modified in this case.